### PR TITLE
fixed add user script

### PR DESCRIPTION
**Pull request by ajloughr-leidos**
# PR Details
## Description

fixed hyphen grep - hyphens within grep statements cause issues. Added a separate grep to check for hyphens 

changed to bash - switched from #!/bin/sh to #!/bin/bash for easier execution

check for mysql-client - added a check to see if mysql-client is installed instead of simply updating and installing every time

fixed minor echo issues and added more feedback 

## Motivation and Context

Script would not execute correctly and had to be run with `bash ./`. Script now runs properly and with simply `./`

## How Has This Been Tested?

Tested many different password combinations including hyphens at beginning, middle, and end of password

## Types of changes

- [X] Defect fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that cause existing functionality to change)

## Checklist:

- [ ] I have added any new packages to the sonar-scanner.properties file
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
[V2XHUB Contributing Guide](https://github.com/usdot-fhwa-OPS/V2X-Hub/blob/develop/Contributing.md) 
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: configuration/amd64/mysql/add_v2xhub_user.sh [hunk 1]
```diff
@@ -12,16 +19,17 @@ read -s -p "Please enter a password: "  PASS
 
 PASS_LENGTH=`echo $PASS | wc -c`
 
-if [ $PASS_LENGTH -ge 8 ] && echo $PASS | grep -q [a-z] && echo $PASS | grep -q [A-Z] && echo $PASS | grep -q [0-9] && echo $PASS | grep -q [\$\!\.\+_-\*@\#\^%\?~]; then
-    echo "Confirm password: "
-    read -s CONF_PASS
+if [ $PASS_LENGTH -ge 8 ] && echo $PASS | grep -q [a-z] && echo $PASS | grep -q [A-Z] && echo $PASS | grep -q [0-9] && ( echo $PASS | grep -q [\$\!\.\+_\*@\#\^%\?~] || echo $PASS | grep -q [-] ); then
+    echo
```
Comment (paulbourelly999) [inline configuration/amd64/mysql/add_v2xhub_user.sh:23]: Can this line be removed now?

File: configuration/amd64/mysql/add_v2xhub_user.sh [hunk 2]
```diff
@@ -12,16 +19,17 @@ read -s -p "Please enter a password: "  PASS
...
+    read -s -p "Confirm password: " CONF_PASS
     while [ $CONF_PASS != $PASS ]; do
-        echo "Passwords do not match. Please re-enter password: "
-        read -s CONF_PASS
+        echo
```
Comment (paulbourelly999) [inline configuration/amd64/mysql/add_v2xhub_user.sh:26]: Can this line be removed now

**Comment by paulbourelly999** [comment]
@ajloughr-leidos One last thing that bothered me but is probably no necessary to fix is I could never get the prompts to separate correctly with new line characters. If we have the time and can fix that as well it might be nice

File: configuration/amd64/mysql/add_v2xhub_user.sh (hunk 1 above)
Comment (ajloughr-leidos) [reply configuration/amd64/mysql/add_v2xhub_user.sh:23]: the echo counteracts the -s and makes the read appear on a new line

File: configuration/amd64/mysql/add_v2xhub_user.sh (hunk 2 above)
Comment (ajloughr-leidos) [reply configuration/amd64/mysql/add_v2xhub_user.sh:26]: the echo counteracts the -s and makes the read appear on a new line

**Comment by ajloughr-leidos** [comment]
The prompts appear on new lines for me. Discussed with Paul and the script is fine as is.

**Review by paulbourelly999 — APPROVED** [review]
Looks good!

**Pull request merged**